README.md

```diff
@@ -1,2 +1,127 @@
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#awesom-books)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 🛠 Built With
+
+- HTML5
+- CSS
+- Javascript ES6
+- Webpack
+
+# Key Features
+
+- This is a To-do List app for adding and removing task from a list
+
+## Learning Objectives
+
+- Flexbox to place elements in the page.
+- Learned how to use webpack
+- Practice of all the knowledge i learned in this module.
+
+# 📖 To-Do List Website
+
+<a name="about-project"></a>
+
+> This is a website for a to-do list tasks.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **It's a website built according to the design provided**
+- **It allows user to add or delete tasks from a to-do list**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://github.com/DannieMo/To-do-list-project.git)
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+**Just clone the project by runing the `git clone git@github.com:DannieMo/To-do-list-project.git` command in your command line, and you are done.**
+
+<!-- AUTHORS -->
+
+## Author
+
+👤 **Daniel Majesty Owiri (Dannie)**
+
+- GitHub: [@dannieMo](https://github.com/DannieMo/)
+- LinkedIn: [Daniel Majesty Owiri](https://linkedin.com/in/daniel-majesty-owiri/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+Give a ⭐️ if you like my effort!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the whole Microverse community for their help and contributions towards this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSED) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Kindly make sure you have added the section `Future Features`, which includes 1-3 potential features to be added. It is a part of the general requirements for every project you will be working on here at Microverse.